manifests/0000_50_olm_12-olm-operators.catalogsource.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
   name: olm-operators
-  namespace: openshift-operator-lifecycle-manager
+  namespace: openshift-marketplace
```

Comment:
I don't think this should've changed to openshift-marketplace, I think it should stay in the olm namespace.

This looks like it's causing the e2e failures as well.

Reply:
That's probably from #871. We might need another variable for that.

Reply:
I will revert that change my PR.

Reply:
On the second thought, there are changes like this across other files so I need to know if we are gonna switch all of them back to olm namespace or just this one specifically.

Reply:
it should just be this one that needs to change, from what I can recall